test/e2e/pkg/client/pod.go

```diff
@@ -32,9 +31,9 @@ func (w *K8sClient) WaitForPodRunningByLabel(label string) (deployed bool, err e
 	for {
 		select {
 		case <-timeout:
-			return false, errors.New("timed out")
+			return false, err
```

Comment:
Does this err ever get set with anything?

Reply:
Done